src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,241 @@
+            break;
+        }
+        isChanged = true;
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + tasks.get(tasks.size() - 1));
+        if (tasks.size() > 1) {
+            System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        } else {
+            System.out.println("Now you have " + tasks.size() + " task in the list.");
+        }
+    }
+
+    //Changes isDone of specified Task as True
+    private static void markAsDone(String input) {
+        int taskIndex;
+
+        try {
+            taskIndex = Integer.parseInt(input);
+        } catch (NumberFormatException e) {
+            System.out.println("OOPS!!! Please list a task number to be marked done: " + input);
+            return;
+        }
+
+        try {
+            tasks.get(taskIndex - 1).markAsDone();
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("OOPS!!! There are only " + tasks.size() + " tasks on the list.");
+            return;
+        }
+        isChanged = true;
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println("  " + tasks.get(taskIndex - 1));
+    }
+
+    private static void removeTask(String input) {
+        int taskIndex;
+
+        try {
+            taskIndex = Integer.parseInt(input);
+        } catch (NumberFormatException e) {
+            System.out.println("OOPS!!! Please list a task number to be deleted: " + input);
+            return;
+        }
+
+        try {
+            Task deletedTask = tasks.get(taskIndex - 1);
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println("OOPS!!! There are only " + tasks.size() + " tasks on the list");
+            return;
+        }
+        System.out.println("Noted. I've removed this task:");
+        System.out.println("  " + tasks.get(taskIndex - 1));
+        tasks.remove(tasks.get(taskIndex - 1));
+        isChanged = true;
+        if (tasks.size() > 1) {
+            System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        } else {
+            System.out.println("Now you have " + tasks.size() + " task in the list.");
+        }
+    }
+
+    //Prints the tasks array
+    private static void displayList() {
+        System.out.println("Here are the tasks in your list:");
+        for (Task task : tasks) {
+            System.out.println((tasks.indexOf(task) + 1) + ". " + task);
+        }
+    }
+
+    //Processes command-line input
+    private static void executeCommand() {
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            String line = in.nextLine();
+            String[] command = line.split(" ", 2);
+            //Identify the specific command by user
+            CommandType commandType = CommandType.COMMAND_UNKNOWN;
+            System.out.println(INDENTLINE);
+            try {
+                commandType = readCommandType(command[0]);
```

Comment:
Maybe you can use a variable to store command[0] so that you wouldnt be confused in the future.